src/constant.js

```diff
@@ -134,8 +134,8 @@ export const CONSTANTS = {
   LIFECYCLE_RECORDING_STARTED: 'Lifecycle History Recording has been started for ',
   APPID_DOESNOT_MATCH: ' passed does not match launched app ',
   INVALID_LIFECYCLE_RECORD: 'Invalid lifecycle record request',
-  WRONG_ERROR_MESSAGE_FORMAT: 'Error Message Format Not Conforming to standard',
-  WRONG_RESPONSE_MESSAGE_FORMAT: 'Unexpected error encountered instead of the anticipated response',
+  WRONG_ERROR_MESSAGE_FORMAT: 'Error message conforming to not supported, but not in the standard format',
```

Comment:
These error logs may not be meaningful to the user still
```
WRONG_ERROR_MESSAGE_FORMAT: Expected "Not supported" error response. Actual response does not conform to the standard error format.
WRONG_RESPONSE_MESSAGE_FORMAT: Unexpected error format encountered in the response.
```

Reply:
corrected